### ISSUE TITLE
Create users not working

### ISSUE THREAD

**Issue by henokzion** [Waziup/Platform#178]
creating users with the provided sample data return with 500 status code
data used
{
  "id": "eb78eed6-d01a-4f69-b562-1f30002c08d9",
  "username": "cdupont",
  "createdTime": "2017-09-01T20:00:00.00Z",
  "firstName": "Corentin",
  "lastName": "Dupont",
  "subservice": "farm1",
  "email": "test@gmail.com",
  "phone": "+390000000",
  "address": "25 Test street",
  "facebook": "test1",
  "twitter": "@Test1",
  "roles": "admin"
}

**Comment by cdupont** [comment]
I have this on server side:
```
api-server_1      | 2018-01-29T16:46:32.988Z - error: Proxy error: TypeError: usersProxy.postUsers is not a function
api-server_1      |     at router.post.req (/usr/src/app/src/proxy.js:156:77)
```

**Comment by cdupont** [comment]
Corrected: adf5494017b6e4827aa7c8fea8d796f8463a8530
Please check
